bundles/org.openhab.binding.loxone/README.md

```diff
@@ -161,6 +161,13 @@ Channel label is defined in the following way:
 *   For controls that belong to a room: `<Room name> / <Control name>`
 *   For controls without a room: `<Control name>`
 
+Channels have the default tags as follows:
+
+*   **Dimmer**: when it belongs to a category of _Lights_ type, the channel will be tagged with _Lighting_ tag. 
+*   **InfoOnlyAnalog**: when it belongs to a category of _Indoor Temperature_ type, it will be tagger with _CurrentTemperature_ tag.
```

Comment:
Just realised InfoOnlyAnalog is a type but doesn't necessarily mean it belongs to temp, can be anything. I have ventilation and humidity configured to it.
I also have indoor temperature type. Are you using infoonlyanalog for all types where the data cannot be changed? I think with temperature you can set the temperature in a room.

Reply:
InfoOnlyAnalog is read only, there is no command to set its value. If you can manipulate your room temperature, it is likely that it will be through the room controller. Binding is not in charge what data will be exposed as InfoOnlyAnalog. Miniserver does. For example it exposes CPU temperature this way.